final-project/static/scripts.js

```diff
@@ -21,22 +21,26 @@ $(function(){
   // If there are product thumbnails on the page, then make them clickable.
   if ( !!(document.querySelector(".clickable-products")) ) {
 
-    // Small TODO: Update this to add a listener on all the products on page.
-    // Small TODO: use the event object to select the appropriate product.
-    // Keep track of the products we have on screen.
-    // BUG: This only listens for clicks on the first product on the page, not all.
-    var productWasClicked = document.querySelector(".clickable-products");
+    // Listen for clicks on all the products on the screen.
+    var productWasClicked = document.querySelectorAll(".clickable-products");
```

Comment:
Why is this named `productWasClicked`?

Reply:
Probably a name I gave it before I wrote the latest assignment code. I will update.